docs/README-migration.md

```diff
@@ -1664,6 +1696,8 @@ This header has been removed and a simplified version of this API has been added
 The standard C headers like stdio.h and stdlib.h are no longer included, you should include them directly in your project if you use non-SDL C runtime functions.
 M_PI is no longer defined in SDL_stdinc.h, you can use the new symbols SDL_PI_D (double) and SDL_PI_F (float) instead.
 
+SDL_bool is now defined as bool, and is 1 byte instead of the size of an int.
```

Comment:
Maybe add here something like `Presence of a conformant stdbool.h is required for C code.` ??

Reply:
I don't think we need that. It'll be pretty obvious if SDL fails to compile, missing stdbool.h, and if it doesn't, people don't need to think about it.

Reply:
OK